repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,23 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
+      {{#parameters}}
+        {
+          "key": "{{key}}",
+          "value": "{{value}}"
+        },
+      {{/parameters}}
+        {
+          "key": "label",
+          "value": "created_by=marathon"
+        },
+        {
+          "key": "label",
+          "value": "dcos_pkg_name=marathon-lb"
+        }
+      ],
```

Comment:
there's an extra comma, it's not valid json, have you tried following these instructions to test out your changes? https://github.com/mesosphere/universe#run-universe-server